src/sentry/search/events/types.py

```diff
@@ -113,6 +114,9 @@ def interval(self) -> float | None:
             return (self.end - self.start).total_seconds()
         return None
 
+    def copy(self) -> "SnubaParams":
```

Comment:
Nit: could use `from __future__ import annotations` and you'll be able to do
```suggestion
    def copy(self) -> SnubaParams:
```

Reply:
Ah I keep forgetting about that ty